Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocColumnRendererBase.cs

```diff
@@ -305,7 +312,7 @@ void IBocColumnRenderer.RenderDataCell (BocColumnRenderingContext renderingConte
     /// This is a template method. Deriving classes must implement <see cref="RenderCellContents"/> to provide the contents of
     /// the table cell (&lt;td&gt;) element.
     /// </remarks>
-    protected virtual void RenderDataCell (
+    protected void RenderDataCell (
```

Comment:
Further clean up: the `RenderDataCell`, `RenderTitleCell`, and `RenderDataColumnDeclaration` methods should be made private. The Argument checks should be moved to the interface implementations since they now become the externally called API.